en/docs/develop/components/api-proxies/policies.md

```diff
@@ -0,0 +1,88 @@
+# Policies
+
+Policies are units of business logic that a user can apply to slightly change the  `Request`, `Response`, or `Error` flow of an API invocation before reaching the backend or the client. For example, you can modify the response of an API invocation by adding a policy to the Response flow to transform the payload from JSON to  XML and to add a header in the response flow. 
+
+Choreo includes a default set of policies covering most of the common use cases. You can attach a policy or multiple policies easily through the Choreo Console. Furthermore, you can attach multiple policies, swap, and rearrange them conveniently.
+
+In Choreo, once you attach a mediation policy to a proxy, the deployment internally is a two-step process.
+
+1. Deployment initiation: generating the interceptor app 
+    If the component you attached the mediation policy to is new, a new repository with the mediation service code based on the attached policies is created and committed. This is referred to as the interceptor app. 
+
+2. Deploying the API
+    Once the deployment initiation is complete, you can provide any configurations required and proceed to deploy. In this step, Choreo builds the generated interceptor app and pushes the docker image to the docker registry. Finally, Choreo deploys the interceptor app with the mediation service and the API Proxy.
+
+ Depending on the flow the mediation policy is attached, the API invocation will undergo the respective behavioral modification as follows: 
+ 
+ ![Request/Response flow](../../../assets/img/api-proxies/policies/request-response-flow.png)
+
+ - In the request path, the requests which pass through the gateway reach the relevant component, and Choreo executes any attached policies to the resource's request path before sending it to the backend. 
+
+- In the response path, the response messages from the backend are first received by the interceptor component, and Choreo executes any mediation policies attached to the `Response` flow or the fault flow. Then the response is forwarded to the client.
+
```

Comment:
New bullet...

If any error occurred during the execution of policies or an internal error occurred, the fault flow will be executed and an error response will be sent to the client.

Reply:
done